### ISSUE TITLE
Use custom tooltip instead of browser tooltip

### ISSUE THREAD

**Issue by thinkh** [Caleydo/taco#91]
At the moment most tooltips are based on Browser tooltips. The problem is that they need a few seconds before they appear. Especially for browsing the diff heatmap it would be helpful if they appear immediatly. Therefore we should replace the browser tooltips with a custom tooltip.

**Comment by doomsayer2** [comment]
Do you mean like I already did once? With a moving div element? We could use a library too but that would add further code and I dont know how to add a specific library.

**Comment by thinkh** [comment]
Yes, but everywhere. You can also try a CSS only solution (e.g., https://codepen.io/cbracco/pen/qzukg).
